README.md

```diff
@@ -11,22 +11,33 @@ If you work a with a git rebase flow, git machete will (vastly!) help you manage
 
 ## Install
 
-Run the following commands to install git machete:
+Installation could be done by following commands:
 
 ```bash
 $ git clone https://github.com/VirtusLab/git-machete.git
 $ cd git-machete
-$ sudo make install
+$ sudo python setup.py install
 ```
 
-Installation could be done by following commands:
+To install you will need python with installed `pip` and `setuptools`.
```

Comment:
Uber-nit: uppercase Python if it refers to software as such rather than shell command

Reply:
OK, I will update.